app/controllers/citizens/property_values_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Citizens
+  class PropertyValuesController < BaseController
+    def show
+      @form = Applicants::PropertyValueForm.new # this is so errors are visible
```

Comment:
Remove the comment. It is wrong. The show action will never show an error. The errors are rendered when the create action renders the show template (and therefore the `show` method is not called)